modules/UI/videolayout/RemoteVideo.js

```diff
@@ -31,6 +31,7 @@ function RemoteVideo(user, VideoLayout, emitter) {
     this.addRemoteVideoContainer();
     this.connectionIndicator = new ConnectionIndicator(this, this.id);
     this.setDisplayName();
+    this.bindHoverHandler();
```

Comment:
Well, bindHoverHandler() depends on addRemoteVideoContainer, which depends on smallvideo constructor. So this is currently not possible.